docs/known_issues.rst

```diff
@@ -33,19 +33,10 @@ Quantities are subclassed from NumPy's `~numpy.ndarray` and in some NumPy
 operations (and in SciPy operations using NumPy internally) the subclass is
 ignored, which means that either a plain array is returned, or a
 `~astropy.units.quantity.Quantity` without units.
-E.g., prior to astropy 4.0 and numpy 1.17::
+E.g.,::
```

Comment:
The workaround for `np.hstack()` is indeed no longer needed and should be deleted.

The other example with the ratio of 3600 seconds and 1 hour is still relevant because it deviates from what is described in this section: https://github.com/astropy/astropy/blob/a3263b683cf91747c062dafe31af994b2848f696/docs/units/quantity.rst#L270 I don't think this example should be removed from the documentation, but perhaps it would be better to move it to the dimensionless quantities section instead. Would you agree?

Reply:
Perhaps the point was that `np.array` also removes the unit...  But worrying a bit less about intent and more about how to make things clearest, it does seem logical to discuss as part of dimensionless quantities. Though I'm still also OK with just removing the lot here.

Reply:
I had another look at the known issues page and I noticed that `np.array` dropping units is already documented here:
https://github.com/astropy/astropy/blob/fd06dc134a464bb8c29e7dfe719dc20167182348/docs/known_issues.rst#L177
That allowed me to simply delete the example in question.

Even better, going through that file I spotted another obsolete known issue which I also deleted.